src/session/sidebar/chat_row.rs

```diff
@@ -236,6 +236,61 @@ fn stringify_message(message: Message) -> String {
         MessageContent::MessageSticker(data) => {
             format!("{} {}", data.sticker.emoji, gettext("Sticker"))
         }
+        MessageContent::MessagePhoto(data) => format!(
+            "{}{}",
+            gettext("Photo"),
+            if data.caption.text.is_empty() {
+                String::new()
+            } else {
+                format!(", {}", data.caption.text)
+            }
+        ),
+        MessageContent::MessageAudio(data) => format!(
+            "{} - {}{}",
+            data.audio.performer,
+            data.audio.title,
+            if data.caption.text.is_empty() {
+                String::new()
+            } else {
+                format!(", {}", data.caption.text)
```

Comment:
Also here and everywhere else below.

Reply:
Done. I put some functions in crate::utils that can be reused. 
Additionally, I escaped the text, else it would not show up when it contains a special character.